app/views/components/_figure.html.erb

```diff
@@ -1,10 +1,14 @@
-<% add_app_component_stylesheet("figure") %>
 <%
+  add_app_component_stylesheet("figure")
   alt ||= ""
   caption ||= ""
   credit ||= ""
+  lang ||= "en"
+  component_helper = GovukPublishingComponents::Presenters::ComponentWrapperHelper.new(local_assigns)
+  component_helper.add_class("app-c-figure")
+  component_helper.set_lang(lang)
```

Comment:
Non-blocking: I think you might not need this line, as the wrapper should set lang automatically. If it doesn't, try initialising it as `local_assigns[:lang] ||= "en"` (might be worth double checking with a test as there currently isn't one).

Reply:
Alternatively since lang wasn't an option on the component previously, you could remove that first line and instead just have `component_helper.set_lang("en")`

Reply:
Good idea, I've used `local_assigns[:lang] ||= "en"` now, thanks.